torchvision/prototype/features/_bounding_box.py

```diff
@@ -158,8 +150,10 @@ def rotate(
             # - points dot matrix (_compute_output_size)
             # Alternatively, we could return new image size by self._F.rotate_bounding_box
             height, width = image_size
-            rotation_matrix = _get_inverse_affine_matrix([0.0, 0.0], angle, [0.0, 0.0], 1.0, [0.0, 0.0])
-            new_width, new_height = _compute_output_size(rotation_matrix, width, height)
+            rotation_matrix = self._F._geometry._get_inverse_affine_matrix(
+                [0.0, 0.0], angle, [0.0, 0.0], 1.0, [0.0, 0.0]
+            )
+            new_width, new_height = self._F._geometry._FT._compute_output_size(rotation_matrix, width, height)
```

Comment:
This sucks but apparently `self._F._geometry._compute_output_size` won't do it. We have to use the `self._F._geometry._FT._compute_output_size` version. Not sure if that's expected and if the two methods can be aligned.

Reply:
It is bad naming issue as two methods are not doing the same thing. 

`self._F._geometry._compute_output_size` refers to https://github.com/pytorch/vision/blob/97e0ea9c6ebc454538b3fa505e1d199547b0feed/torchvision/transforms/functional.py#L363-L383
and is used in `F.resize` to compute output size given specific options.

`self._F._geometry._FT._compute_output_size` refers to https://github.com/pytorch/vision/blob/97e0ea9c6ebc454538b3fa505e1d199547b0feed/torchvision/transforms/functional_tensor.py#L613-L643
and is used in `F.rotate` with `expand=True`. I suggest to rename them for clarity